habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -386,6 +386,290 @@ def step(self, *args, is_last_action, **kwargs):
             return {}
 
 
+class Controller:
+    """
+        A controller that takes the input of the waypoints and
+        produces the velocity command.
+    """
+    def __init__(self, v_max_default, w_max_default, track_yaw=True):
+        # If we want track yaw or not
+        self.track_yaw = track_yaw
+
+        # Parameter of the controller
+        self.v_max = v_max_default
+        self.w_max = w_max_default
+
+        self.lin_error_tol = self.v_max / 120
```

Comment:
What is 120 here?

Reply:
Thanks for this! This has been fixed to allow users to pass the argument